core/src/test/scala/org/apache/spark/DriverSuite.scala

```diff
@@ -28,31 +28,30 @@ import org.apache.spark.util.Utils
 
 class DriverSuite extends FunSuite with Timeouts {
 
+  // Regression test for SPARK-530: "Spark driver process doesn't exit after finishing"
```

Comment:
Since you're touching this, I kinda like the approach of adding the bug number to the `test()` call that I've seen in other places.